test/hotspot/jtreg/compiler/jvmci/compilerToVM/IterateFramesNative.java

```diff
@@ -48,19 +44,21 @@
  *                   -XX:+UnlockExperimentalVMOptions -XX:+EnableJVMCI
  *                   -XX:+DoEscapeAnalysis -XX:-UseCounterDecay
  *                   compiler.jvmci.compilerToVM.IterateFramesNative
+ * @run main/othervm -Xbatch -Xcomp -Xbootclasspath/a:.
+ *                   -XX:+UnlockDiagnosticVMOptions -XX:+WhiteBoxAPI
+ *                   -XX:+UnlockExperimentalVMOptions -XX:+EnableJVMCI
+ *                   -XX:+DoEscapeAnalysis -XX:-UseCounterDecay
```

Comment:
Are the `-XX:+DoEscapeAnalysis -XX:-UseCounterDecay` options really needed?

Reply:
Probably copied them from somewhere, I think they're irrelevant and will remove them.